source/shared/net/guides/deciding-wallet.rst

```diff
@@ -1,79 +1,100 @@
+- **Backup of the Mobile Wallet**: You create a backup of your accounts, identities, addresses, and private keys by exporting them to a file from the Mobile Wallet. If you lose your phone or upgrade to a new phone, you can use the file to gain access to your accounts and identities. Alternatively, if you're using an iPhone, you can make a backup from iCloud. It's vital that you keep the password to the backup file safe. Anyone with access to the file can gain access to your crypto assets. For more information, see :ref:`Explore the *More* page in the Mobile Wallet <explore-more>`.
 
-.. insert link to topic Learn more about the Ledger, when created
+.. Warning::
+   You are solely responsible for keeping your assets secure regardless of which wallet you choose to use. You must never share your private keys, PIN codes, passwords, recovery phrases, Ledgers, or mobile devices with anyone.
+
+What are the requirements for each wallet?
+==========================================
+
+- **Mobile Wallet**: you’ll need an iPhone running iOS 13 or later or an Android phone running Android 8 or later.
+
+- **Desktop Wallet**: you’ll need a computer running Windows, macOS, or Linux, and you’ll need a Ledger Nano S. You’ll also have to run a node, and your computer must meet the hardware requirements for this.
 
 Comparison
 ==========
 
 The following table lists the major features of each wallet.
 
 .. list-table::
-   :widths: 20 20 20
+   :widths: 20 20
    :header-rows: 1
 
-   * - Features
-     - Desktop Wallet
-     - Mobile Wallet
-   * - **Secured by password and biometrics**
-     - Password
-     - Password and biometrics
-   * - **Encrypted**
-     - Yes
-     - Yes
-   * - **Create backup of identities and accounts**
-     - Yes. The backup includes accounts, identities, and the address book.
-     - Yes. The backup includes accounts, identities, address book, and account keys.
-   * - **Storage of private keys**
-     - Stored on the Ledger that is secured by a PIN code and backed up by recovery phrase.
-     - Stored in the wallet.
-   * - **Create and manage single signature accounts**
-     - Yes
-     - Yes
-   * - **Create and manage multi signature accounts**
-     - Yes
-     - Yes
-   * - **Create identities**
-     - Yes
-     - Yes
-   * - **Create and manage baker accounts**
-     - Yes
-     - The Concordium Client can be used to manage baker accounts. Rewards are visible in the wallet.
-   * - **Run a node**
-     - Requirement
-     - Not needed
+   *  - **Desktop Wallet**
+      - **Mobile Wallet**
+   *  - Secured by password
+      - Secured by password and biometrics
+   *  - Encrypted
+      - Encrypted
+   *  - Backup includes accounts, identities, and the address book.
```

Comment:
Is this right. Backup does not include account keys, right? Will people be able to distiguish between accounts and account private keys? So the main thing is probably identities

Reply:
I misread this the first time. Here's my new take after re-reading it:

What Stefan said about the identities being the main thing is right I think, as they can't be restored using only the Ledger, although I guess the same goes for the address book. Accounts technically can be restored from the Ledger and recovery phrase alone, but that doesn't contain the given account names, so there's sort of "two parts" to backing them up.

So maybe we should phrase it like the export contains names and addresses for the accounts, the identities and the address book. In combination with the Ledger, or a recovery phrase and a Ledger, the export can used to recover everything from the the desktop wallet. 

I don't know if it makes more sense now, @LotteKh?